### PR TITLE
Add dockerfiles

### PR DESCRIPTION
**Pull request by yaamai**

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -1,46 +1,88 @@
...
+        "cp script/uninstall.sh release"
+      ]
+    },
+    {
+      "name": "github release",
+      "image": "plugins/github-release",
+      "settings": {
+        "api_key": {"from_secret": "github_api_key"},
+        "files": "release/*",
+	    "checksum": ["sha256"]
```
Comment (rsp9u) [inline .drone.star:36]: This line's indentation contains mixed spaces and tabs.

File: README.md [hunk 2]
```diff
@@ -83,6 +83,21 @@ This is a minimal and lightweight virtual machine manager.
 # curl -Lo - https://github.com/rsp9u/minivmm/releases/latest/download/install.sh | sh -
 ```
 
+### Standalone with Docker
+```
+$ sudo podman run \
```
Comment (rsp9u) [inline README.md:88]: Docker? podman?

File: .drone.star (hunk 1 above)
Comment (rsp9u) [reply .drone.star:36]: Here seems to have been like this before. Could you fix this?

Comment (yaamai) [reply .drone.star]: fixed

File: README.md (hunk 2 above)
Comment (yaamai) [reply README.md]: fixed

**Pull request merged**